docker-postgis/README.md

```diff
@@ -0,0 +1 @@
+from https://github.com/appropriate/docker-postgis/tree/master/10-2.5/alpine
```

Comment:
should the name of the folder be changed to `docker` or `docker-scripts`? Seems like it's not only about postgis.

Reply:
DONE